app/inbox/models.py

```diff
@@ -1,30 +1,23 @@
 from django.db import models
+from economy.models import SuperModel
 from django.utils.translation import gettext_lazy as _
 from django.contrib.auth import get_user_model
-from django.forms.models import model_to_dict
 
 
-class Notification(models.Model):
-    created_on = models.DateTimeField(auto_now_add=True, editable=False)
-    modified_on = models.DateTimeField(auto_now=True)
-    CTA_URL = models.URLField(max_length=255, null=True)
-    CTA_Text = models.CharField(max_length=255)
-    message_html = models.CharField(max_length=255, null=True, help_text=_("Html message"))
+class Notification(SuperModel):
+
+    cta_url = models.URLField(max_length=255, blank=True)
+    cta_text = models.CharField(max_length=255)
```

Comment:
restrict by possible choices - defining the choices and individual elements for `choices`.

```python
SOME_OPTION = 'SO'
ANOTHER_OPTION = 'AO'

NOTIFICATION_TYPES = (
    (SOME_OPTION, 'some_option'),
    (ANOTHER_OPTION, 'another_option'),
)

cta_text/notification_type = models.CharField(max_length=2, choices=NOTIFICATION_TYPES, default=SOME_OPTION)
```

Reply:
I think we can create 
'sent_tip' 
'sent_kudos'

'worker_started'
'worker_approved'
'worker_declined'
'worker_removed'
'work_submitted'
...
Or even better we can follow this options too https://github.com/gitcoinco/web/blob/master/app/dashboard/models.py#L1500

Reply:
I will put in all the options I guess.